pkg/domain/infra/abi/manifest.go

```diff
@@ -0,0 +1,102 @@
+// +build ABISupport
+package abi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	buildahUtil "github.com/containers/buildah/util"
+	"github.com/containers/image/v5/docker"
+	"github.com/containers/image/v5/transports/alltransports"
+	libpodImage "github.com/containers/libpod/libpod/image"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/util"
+
+	"github.com/pkg/errors"
+)
+
+// ManifestCreate implements logic for creating manifest lists via ImageEngine
+func (ir *ImageEngine) ManifestCreate(ctx context.Context, names, images []string, opts entities.ManifestCreateOptions) error {
+	fullNames, err := buildahUtil.ExpandNames(names, "", ir.Libpod.SystemContext(), ir.Libpod.GetStore())
+	if err != nil {
+		return errors.Wrapf(err, "error encountered while expanding image name %q", names)
+
+	}
+	imageID, err := libpodImage.CreateManifestList(ir.Libpod.ImageRuntime(), *ir.Libpod.SystemContext(), fullNames, images, opts.All)
+	if err == nil {
+		fmt.Printf("%s\n", imageID)
+	}
+	return err
+}
+
+// ManifestInspect returns the content of a manifest list or image
+func (ir *ImageEngine) ManifestInspect(ctx context.Context, name string) error {
+	dockerPrefix := fmt.Sprintf("%s://", docker.Transport.Name())
+	_, err := alltransports.ParseImageName(name)
+	if err != nil {
+		_, err = alltransports.ParseImageName(fmt.Sprintf("%s%s", dockerPrefix, name))
+		if err != nil {
+			return errors.Errorf("invalid image reference %q", name)
+		}
+	}
+	image, err := ir.Libpod.ImageRuntime().New(ctx, name, "", "", nil, nil, libpodImage.SigningOptions{}, nil, util.PullImageMissing)
+	if err != nil {
+		return errors.Wrapf(err, "reading image %q", name)
+	}
+
+	list, err := image.InspectManifest()
+	if err != nil {
+		return errors.Wrapf(err, "loading manifest %q", name)
+	}
+	buf, err := json.MarshalIndent(list, "", "    ")
+	if err != nil {
+		return errors.Wrapf(err, "error rendering manifest for display")
+	}
+	fmt.Printf("%s\n", buf)
```

Comment:
```suggestion
return buf, nil
```
This needs to be returned to caller, not printed at this layer.